spec/OpenLineage.md

```diff
@@ -111,3 +110,7 @@ Custom facets can be promoted to the standard by including them in the spec.
 - **schema**: Captures the schema of the dataset
 
 - **dataSource**: Captures the Database instance containing this datasets (ex: Database schema. Object store bucket, ...)
+
+#### Input Dataset Facets
```

Comment:
_Minor_: Should we add a comment communicating that no standard inputs / outputs? facets are yet defined?

Reply:
added